lib/mix/lib/mix/tasks/do.ex

```diff
@@ -24,20 +24,10 @@ defmodule Mix.Tasks.Do do
   end
 
   defp gather_commands(args) do
-    gather_commands args, [], []
-  end
-
-  defp gather_commands([h | t], current, acc) when binary_part(h, byte_size(h), -1) == "," do
-    part    = binary_part(h, 0, byte_size(h) - 1)
-    current = Enum.reverse([part | current])
-    gather_commands t, [], [current | acc]
-  end
-
-  defp gather_commands([h | t], current, acc) do
-    gather_commands t, [h | current], acc
-  end
-
-  defp gather_commands([], current, acc) do
-    Enum.reverse [Enum.reverse(current) | acc]
+    args
+    |> Enum.join(" ")
+    |> String.split(",")
+    |> Enum.map(&String.strip(&1))
```

Comment:
Probably better start using `String.trim`. :bowtie:
Do we want to trim all Unicode whitespaces (newlines, etc.) here?